src/docs/getting-started/README.md

```diff
@@ -2,19 +2,25 @@
 
 In this article, we are going to see how easy it is to create a CMS Web application using the NuGet packages provided by Orchard Core.
 
+!!! tip
+    Do you just want to test drive Orchard Core, without creating a .NET application? Check out [this guide](test-drive-orchard-core.md).
+
 ## Create an Orchard Core CMS application
 
-In Visual Studio, create a new empty .NET Core web application. Ex: `Cms.Web`. Do not check "Place solution and project in the same directory", because later when you create modules and themes you will want them to live alongside the web application within the solution.
+!!! note
+    This guide follows a manual process so you can have a deeper understanding of what's happening. You can also use our [code generation templates](templates/README.md) for a quicker start if you create a brand new application. Check out [this guide](../guides/create-cms-application/README.md).
```

Comment:
This overlaps with the `create-cms-application` page. I wasn't sure how to solve this.